roles/lib_dyn/library/dyn_record.py

```diff
@@ -0,0 +1,274 @@
+# This is a frustrating lookup dict to allow mapping to the RECORD_PARAMS
+# dict so we can lookup other values in it efficiently
+
+def get_record_type(record_key):
+    '''Get the record type represented by the keys returned from get_any_records.'''
+    return record_key.replace('_records', '').upper()
+
+def get_record_key(record_type):
+    '''Get the key to look up records in the dictionary returned from get_any_records.'''
+    return record_type.lower() + '_records'
+
+def get_any_records(module, node):
+    '''Get any records for a given node'''
+    # Lets get a list of the A records for the node
+    try:
+        records = node.get_any_records()
+    except dyn.tm.errors.DynectGetError as error:
+        if 'Not in zone' in str(error):
+            # The node isn't in the zone so we'll return an empty dictionary
+            return {}
+        else:
+            # An unknown error happened so we'll need to return it.
+            module.fail_json(msg='Unable to get records',
+                             error=str(error))
+
+    # Return a dictionary of the record objects
+    return records
+
+def get_record_values(records):
+    '''Get the record values for each record returned by get_any_records.'''
+    # This simply returns the values from a dictionary of record objects
+    ret_dict = {}
+    for key in records.keys():
+        record_type = get_record_type(key)
+        record_value_param = RECORD_PARAMS[record_type]['value_param']
+        ret_dict[key] = [getattr(elem, record_value_param) for elem in records[key]]
+    return ret_dict
+
+def main():
+    '''Ansible module for managing Dyn DNS records.'''
+    module = AnsibleModule(
+        argument_spec=dict(
+            state=dict(required=True, choices=['present', 'absent', 'list']),
+            customer_name=dict(default=os.environ.get('DYNECT_CUSTOMER_NAME', None), type='str'),
+            user_name=dict(default=os.environ.get('DYNECT_USER_NAME', None), type='str', no_log=True),
+            user_password=dict(default=os.environ.get('DYNECT_PASSWORD', None), type='str', no_log=True),
+            zone=dict(required=True),
+            record_fqdn=dict(required=False),
+            record_type=dict(required=False, choices=[
+                'A', 'AAAA', 'CNAME', 'PTR', 'TXT']),
+            record_value=dict(required=False),
+            record_ttl=dict(required=False, default=0, type='int'),
+        ),
+        required_together=(
+            ['record_fqdn', 'record_value', 'record_ttl', 'record_type']
+        )
+    )
+
+    if IMPORT_ERROR:
+        module.fail_json(msg="Unable to import dyn module: https://pypi.python.org/pypi/dyn",
+                         error=IMPORT_ERROR)
+
+    # Start the Dyn session
+    try:
+        # dyn_session is needed to access the other objects but rarely gets
+        # used on its own.
+        #pylint: disable=unused-variable
+        dyn_session = DynectSession(module.params['customer_name'],
```

Comment:
instead of dyn_session, pylint wants you to use _ as a variable.  That will pass the check and you can remove the pylint disable